src/test/java/com/github/javafaker/MedicalTest.java

```diff
@@ -0,0 +1,41 @@
+package com.github.javafaker;
+
+import org.junit.Test;
+
+import static com.github.javafaker.matchers.MatchesRegularExpression.matchesRegularExpression;
+import static org.junit.Assert.assertThat;
+
+public class MedicalTest extends AbstractFakerTest {
+
+    @Test
+    public void testMedicineName() {
+        assertThat(faker.medical().medicineName(), matchesRegularExpression("([\\w']+\\.?( )?){2,5}"));
+    }
+
+    @Test
+    public void testDiseaseName() {
+        assertThat(faker.medical().diseaseName(), matchesRegularExpression("([\\w']+\\.?( )?){2,8}"));
+    }
+
+    @Test
+    public void testHospitalName() {
+        assertThat(faker.medical().hospitalName(), matchesRegularExpression("([\\w',]+\\.?.\\(?[\\w']\\)?( )?){2,10}"));
+    }
+
+    @Test
+    public void testSymptom() {
+        assertThat(faker.medical().symptoms(), matchesRegularExpression("([\\w']+\\.?.\\(?[\\w']\\)?( )?){2,5}"));
+    }
+
+
+    @Test
+    public void testSymptomStatic() {
```

Comment:
Should this make a call to the faker class?

Reply:
Hi @codingricky 
All the static test cases were kind of special cases (containing '(' OR ',')
But are now taken care of in regular test cases RegEx itself (eg. testHospital & testSymptom).
They can be removed, just added them for testing special cases.
Please suggest if i should remove them